### DISCUSSION TITLE
Alpine Expression Error: checkoutInformationForm is not defined

### DISCUSSION THREAD

**Discussion by tobiv** [duncanmcclean/simple-commerce#979] [Help] [answered]
Hi! I'm building a shop on top of an existing Statamic site, piecing together with parts of the SC starter kit. On the `checkout/information` page, I'm getting lots of Alpine errors, essentially on every `x-data`/`x-model` attribute it says not defined. Haven't really changed the starter kit templates other than wording/layout/styles. Maybe I'm missing some javascript? Other than installing and loading alpine, I couldn't find any more in the templates though. Let me know if I can provide any more information.

**Comment by duncanmcclean** [comment]
Have you copied across this bit?

https://github.com/duncanmcclean/sc-starter-kit/blob/main/resources/views/checkout/information.antlers.html#L122-L145

**Reply by tobiv** [reply]
Yes, I have. I just noticed another thing though: `{{ json_regions }}` (top of the script) doesn't output anything:

    window.regions = ;

**Reply by duncanmcclean** [reply, marked as answer]
Ah, that's a custom tag you'll need to copy across into your app. Just copy the contents of this file into `app/Tags/JsonRegions.php` and it should output some JSON.

https://github.com/duncanmcclean/sc-starter-kit/blob/main/app/Tags/JsonRegions.php

**Reply by tobiv** [reply]
Seems to have fixed the alpine error. Thank you!
